public/casestudies/adeo.yml

```diff
@@ -0,0 +1,283 @@
+                                <id>install @asyncapi/generator globally</id>
+                                <goals>
+                                    <goal>npm</goal>
+                                </goals>
+                                <configuration>
+                                    <arguments>install @asyncapi/generator@${asyncapi.generator.version}</arguments>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <version>1.6.0</version>
+
+                        <executions>
+                            <execution>
+                                <id>execute-generation</id>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <phase>generate-resources</phase>
+
+                                <configuration>
+                                    <!-- Access binary file in node_modules because it doesn't work on windows otherwise. -->
+                                    <executable>${node.modules.installation.path}/${ag.binary.name}</executable>
+                                    <commandlineArgs>
+                                        ${project.basedir}/src/docs/asyncapi/asyncapi.yaml @asyncapi/html-template@${asyncapi.htmltemplate.version} -p sidebarOrganization=byTags -p
+                                        version=${project.version} -o ${asyncapi.generation.dir}
+                                    </commandlineArgs>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>execute-internal-generation</id>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <phase>generate-resources</phase>
+
+                                <configuration>
+                                    <!-- Access binary file in node_modules because it doesn't work on windows otherwise. -->
+                                    <executable>${node.modules.installation.path}/${ag.binary.name}</executable>
+                                    <commandlineArgs>
+                                        ${project.basedir}/src/docs/asyncapi/asyncapi-internal.yaml @asyncapi/html-template@${asyncapi.htmltemplate.version} -p sidebarOrganization=byTags -p
+                                        version=${project.version} -o ${asyncapi.generation.dir}/internal
+                                    </commandlineArgs>
+                                </configuration>
+                            </execution>
```

Comment:
you can remove this part that do the same things as below but for our internal doc

Reply:
sorry but can you specify code line....I'm not joking....just when I see maven and XML, I just see one big dot and no differences 😄

Reply:
from line 171 to line 186 ;) (like shown in the github snippet)